test/src/test/java/hudson/util/AtomicFileWriterPerfTest.java

```diff
@@ -24,7 +24,7 @@
      * <strong>really</strong> bad performance regressions.
      */
     @Issue("JENKINS-34855")
-    @Test(timeout = 30 * 1000L)
+    @Test(timeout = 80 * 1000L)
```

Comment:
Why? Has this worsened performance by ~2x?

Reply:
Worse than that, around ~30+ for that very case. I just reduced to 50 seconds, since it actually takes ~35 seconds in CI.
 
But as explained in the [commit log](https://github.com/jenkinsci/jenkins/pull/3170/commits/f555cb6e7a0d4199a4417841e7b42e7f61786c22), I still think the overall change is an improvement. Before, we were quicker, but unsafe, is the whole point of this... :-|